src/utils.ts

```diff
@@ -45,3 +45,7 @@ export function splitChildren(children: React.ReactNode): PlotChildren {
   }
   return { invalidChild, lineSeries, axis, heading, legend };
 }
+
+export function camelToKebab(string: string) {
+  return string.replace(/([a-z0-9]|(?=[A-Z]))([A-Z])/g, '$1-$2').toLowerCase();
```

Comment:
```suggestion
  // eslint-disable-next-line ...
  return string.replace(/([a-z0-9]|(?=[A-Z]))([A-Z])/g, '$1-$2').toLowerCase();
```

Reply:
I wasn't able to find how to do it with replaces, but here's the link if someone else wants it https://2ality.com/2017/05/regexp-named-capture-groups.html